website/templates/project/project.mako

```diff
@@ -276,7 +276,11 @@
 <div class="row">
     <div class="col-xs-12">
         <div class="pp-notice m-b-md p-md clearfix">
-            This project represents a preprint. <a href="http://help.osf.io/m/preprints">Learn more</a> about how to work with preprint files.
+            This project represents a preprint.
+            % if node['has_preprint_moderated']:
+                The preprint is currently in ${node['preprint_state']} state.
```

Comment:
This might be more confusing than helpful. If we're displaying the state, we should maybe include a description of what it means, like we do on the preprint status banner.

Reply:
do you think a popover message might be a good idea or just add to message text